samples/local_attestation/common/attestation.cpp

```diff
@@ -134,32 +142,24 @@ bool Attestation::attest_local_report(
 
     // Check the enclave's product id and security version
     // See enc.conf for values specified when signing the enclave.
-    if (parsed_report.identity.product_id[0] != 1)
```

Comment:
product_id can be derived from claims.

Reply:
I am not sure what you mean by this. I replaced the identity with the claims method.

Reply:
@yentsanglee Can I resolve this issue?